amd64/root/start.sh

```diff
@@ -40,6 +44,23 @@ if [ "$DECONZ_VNC_MODE" != 0 ]; then
   
   # Export VNC display variable
   export DISPLAY=$DECONZ_VNC_DISPLAY
+  
+  if [ "$DECONZ_NOVNC_PORT" = 0 ]; then
+    echo "[marthoc/deconz] noVNC Disabled"
+  else
+    if [ "$DECONZ_NOVNC_PORT" -lt 6080 ]; then
+      echo "[marthoc/deconz] ERROR - NOVNC port must be 6080 or greater!"
+      exit 1
+    fi
+
+    # Generate SSL certificate
+    openssl req -x509 -nodes -newkey rsa:2048 -keyout /root/.vnc/novnc.pem -out /root/.vnc/novnc.pem -days 365 -subj "/CN=deconz"
+
```

Comment:
Hi, have a look at my suggestion [here in PR #341](https://github.com/marthoc/docker-deconz/pull/341#issuecomment-808901881).
If the `.vnc` directory would be placed inside the mounted volume (i.e. `/root/.local/share/dresden-elektronik/deCONZ/.vnc`) the certificate could be placed there and be issued by the user as well.

Reply:
I don't think that we want to mess with that directory as it is proprietary to deCONZ. Instead we should use a directory like /root/.vnc (which is what @phdelodder has chosen).

Reply:
You are right. I forgot that this is more a deCONZ volume than a docker-deconz ...

Actually this is the default the `tigervncserver` script is using.

But shouldn't it be possible to setup `/root/.vnc` as mounted docker volume?

Reply:
I'm going to add some checks to see if the file exists. In a near future I'm going to make it possible to select your own certificate. 

For the mounting `/root/.vnc/` docker volume, would require to rework the current password storage and certificate storage. For those improvements there is going to be a different PR later on.

Reply:
Maybe I don't get the concept of volumes right ... but if `/root/.vnc/` is mounted why does it have impact on the script?

Additionally it won't be a bad idea to store the path information in a variable. In the vnc.config file it is `vncUserDir`.

If you will add a check for the existence of a certificate you might consider validating the file. Eventually it might be in use for a year: 🥳 
```sh
openssl x509 -noout -in /root/.vnc/novnc.pem -dates
```

Reply:
The `/root/.vnc/passwd` is currently used to store the password, if enter through a secret that would expose the value of the secret and not so ideal. I'm planning on a review of all those things in the near future and put them in a future PR. 

The openssl date validation I'll look in to, perhapse it's also for a next PR. If it already exists will be added in the current one.